itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcrossInstances.java

```diff
@@ -1097,6 +1100,8 @@ public Boolean apply(@Nullable CallerArguments args) {
 
     // Retry with different dump should fail.
     replica.loadFailure(replicatedDbName, tuple2.dumpLocation);
+    CommandProcessorResponse ret = replica.runCommand("REPL LOAD " + replicatedDbName + " FROM '" + tuple2.dumpLocation + "'");
```

Comment:
Shall remove the duplicate verification step of REPL LOAD command with replica.loadFailure.

Reply:
done